macros/src/lib.rs

```diff
@@ -222,6 +223,7 @@ fn derive_storage_keys_inner(struct_def: TokenStream2) -> TokenStream2 {
                 &[ #ident_list ]
             };
 
+            #[allow(dead_code)]
```

Comment:
could we leave `VALUES` without `#[allow(dead_code)]`? It can help to spot if any of the keys are not being used

Reply:
yeah, sure, at least `VALUES` should be used